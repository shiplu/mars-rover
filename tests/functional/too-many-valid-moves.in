10 10
1 1 E
MMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLMLMMMMMMMMMLML
