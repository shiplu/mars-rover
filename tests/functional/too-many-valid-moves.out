1 1 E OPERATIONAL
